modules/client/src/main/scala/ch/epfl/bluebrain/nexus/admin/client/AdminClient.scala

```diff
@@ -73,14 +77,18 @@ object AdminClient {
         projectClient(requestFrom(credentials, path)).recoverWith { case e => recover(e, path) }
       }
 
-      override def getProjectAcls(name: ProjectReference)(
+      override def getProjectAcls(name: ProjectReference, parents: Boolean = false, self: Boolean = false)(
           implicit credentials: Option[OAuth2BearerToken]): Future[FullAccessControlList] = {
-        val path = name.value / "acls"
-        aclsClient(requestFrom(credentials, path)).recoverWith { case e => recover(e, path) }
+        val path  = name.value / "acls"
+        val query = Query("parents" -> parents.toString, "self" -> self.toString)
+        aclsClient(requestFrom(credentials, path, Some(query))).recoverWith { case e => recover(e, path) }
       }
 
-      private def requestFrom(credentials: Option[OAuth2BearerToken], path: Path) = {
-        val request = Get(config.baseUri.append(path))
+      private def requestFrom(credentials: Option[OAuth2BearerToken], path: Path, query: Option[Query] = None) = {
```

Comment:
Since you have the implicit credentials already, why not to have the signature like
```scala
private def requestFrom(path: Path, query: Option[Query] = None)(implicit cred: Option[OAuth2BearerToken])
```